_posts/2021-03-15-changelog-143.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post title: "IntelliJ Rust Changelog #143"
+date: 2021-03-15 17:00:00 +0300
+---
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6564 -->
+* Provide initial support for [function-like](https://doc.rust-lang.org/reference/procedural-macros.html#function-like-procedural-macros) procedural macros.
+  Now the plugin can expand such procedural macro calls and as a result they
+  automatically get a bunch of some features available for declarative macros:
+  highlighting, name resolution, restricted completion, `Show macro expansion` intentions and others.
+  Note, it's only initial implementation, so it may work in unexpected way in some cases.
+  The feature is disabled for default for now, to turn if on,
+  you should enable `org.rust.cargo.evaluate.build.scripts` and `org.rust.macros.proc` experimental features.
```

Comment:
Let's add a note that after enabling these features a user should manually refresh the cargo projects (or restart the IDE)

Reply:
Thanks!
Maybe it's worth just adding the corresponding listeners into `org.rust.cargo.project.model.impl.CargoProjectsServiceImpl`?